js_modules/dagster-ui/packages/ui-core/src/asset-data/AssetHealthDataProvider.tsx

```diff
@@ -35,21 +35,33 @@ function init() {
           },
         });
       } else {
-        healthResponse = {data: {assetNodes: [] as AssetHealthFragment[]}};
+        healthResponse = {
+          data: {
+            assetsOrError: {nodes: [] as AssetHealthFragment[], __typename: 'AssetConnection'},
+          },
+        };
       }
 
       const {data} = healthResponse;
+      switch (data.assetsOrError.__typename) {
```

Comment:
TODO - actually handle these errors. Is there a standard pattern for this kind of thing?

Reply:
I think probably just show a toast with `showCustomAlert`